packages/@aws-cdk/aws-appconfig-alpha/README.md

```diff
@@ -121,6 +121,8 @@ configuration data is syntactically and semantically correct. You can create val
 Lambda function.
 See [About validators](https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-creating-configuration-and-profile.html#appconfig-creating-configuration-and-profile-validators) for more information.
 
+When you import a JSON Schema validator from a file, you can pass in a relative path.
```

Comment:
Can you please update [here](https://github.com/aws/aws-cdk/blob/78b34accfa1cba88cc412b04df42ea5819c2cf4c/packages/%40aws-cdk/aws-appconfig-alpha/README.md?plain=1#L96) as well?
```
* `fromFile` - Defines the hosted configuration content from a file (you can specify a relative path).